conceptnet5/readers/emoji.py

```diff
@@ -0,0 +1,49 @@
+import xml.etree.ElementTree as et
+from conceptnet5.edges import make_edge
+from conceptnet5.formats.msgpack_stream import MsgpackStreamWriter
+from conceptnet5.uri import Licenses, concept_uri
+
+def is_sentence(text):
+	if(' ' in text and not '|' in text):
+		return True
+	else:
+		return False
+
+def amt_words(text):
+	return text.count('|')+1
+
+def strip_words(text):
+	res =  text.split('|')
+	for i in range(len(res)):
```

Comment:
Why eliminate all spaces?

If you want to eliminate the spaces that surround the text, use `res[i].strip()` instead of `res[i].replace(" ", "")`.

Also, instead of writing out the for loop that replaces one string at a time, you can use a list comprehension:

    res = [item.strip() for item in res]

Finally, these extra spaces are showing up around the vertical bars. As long as they're always there, you could split on the entire string `" | "` instead of `"|"`.